tests/locales/Xamarin.Android.Locale-Tests/Xamarin.Android.Locale-Tests.csproj

```diff
@@ -18,12 +18,10 @@
     <AndroidManifest>Properties\AndroidManifest.xml</AndroidManifest>
     <AndroidSupportedAbis>armeabi-v7a;x86</AndroidSupportedAbis>
     <AndroidUseSharedRuntime>False</AndroidUseSharedRuntime>
-    <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
+    <TargetFrameworkVersion>v8.1</TargetFrameworkVersion>
+    <AndroidUseLatestPlatformSdk>true</AndroidUseLatestPlatformSdk>
   </PropertyGroup>
```

Comment:
What is the reasoning for this change? Wouldn't it limit the tests, when nearly everything would target v8.1?

Reply:
This is just to bring parity with other tests - note however line 22, it renders line 21 ineffective by choosing whatever the latest platform is. The test projects must be somehow made to use the same project options, but that's a material for a different PR.